include/deal.II/lac/solver_cg.h

```diff
@@ -172,6 +173,19 @@ namespace LinearAlgebra
  * run the updates on the vectors according to a variant presented in
  * Algorithm 2.2 of @cite Chronopoulos1989 (but for a preconditioner), whereas
  * the operation after the loop performs a total of 7 reductions in parallel.
+ *
+ * <h3>Preconditioned residual</h3>
+ *
+ * @p AdditionalData allows you to choose between using the explicit
+ * or implicit residual as stopping condition for the interative
+ * solver. This behavior can be overridden by using the flag
```

Comment:
```suggestion
 * @p AdditionalData allows you to choose between using the explicit
 * or implicit residual as stopping condition for the iterative
 * solver. This behavior can be overridden by using the flag
```
In addition, I would suggest to spend one sentence on what explicit and implicit residuals are: The former explicitly calculates `b - A x^{(k)}` with an additional matrix-vector product, whereas the latter uses the result of the matrix-vector product already computed in other steps of the algorithm to derive the residual by a mere vector update.